File: SCH/SCH.Client/src/app/enums/sch-error-number.ts
/**
 * Global error codes for the application
 * These codes should match the C# SCHErrorNumber enum in SCH.Shared.Exceptions
 */
export enum SCHErrorNumber {
  /**
   * General/unknown error
   */
  None = 0,

  /**
   * Concurrency conflict - record was modified by another user
   */
  ConcurrencyConflict = 1
}

File: SCH/SCH.Shared/Exceptions/SCHErrorNumber.cs
namespace SCH.Shared.Exceptions
{
    /// <summary>
    /// Global error codes for the application
    /// These codes should match the TypeScript SCHErrorNumber enum in the client
    /// </summary>
    public enum SCHErrorNumber
    {
        /// <summary>
        /// General/unknown error
        /// </summary>
        None = 0,

        /// <summary>
        /// Concurrency conflict - record was modified by another user
        /// </summary>
        ConcurrencyConflict = 1
    }
}
